apps/studio/components/interfaces/Billing/Payment/AddPaymentMethodForm.tsx

```diff
@@ -111,7 +111,7 @@ const AddPaymentMethodForm = ({
   }
 
   return (
-    <form onSubmit={handleSubmit}>
```

Comment:
Removed the form here, as otherwise form on a higher level component are broken and submitted as well causing issues, we dont wanna have forms inside these inlined components